### PR TITLE
Prepare build scripts for upload and release on JCenter and MavenCent…

### PR DESCRIPTION
**Pull request by chris-yv**
…ral.

@stevelilly-youview please review.

### CODE REVIEW

File: snapper-samples/build.gradle [hunk 1]
```diff
@@ -22,5 +22,5 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'com.android.support:appcompat-v7:23.0.1'
```
Comment (stevelilly-youview) [inline snapper-samples/build.gradle]: How does it work if the project importing snapper uses a slightly different version of appcompat?

**Comment by stevelilly-youview** [comment]
Review done.

File: snapper-samples/build.gradle (hunk 1 above)
Comment (chris-yv) [inline snapper-samples/build.gradle]: The samples aren't included in the released binary, you only get them from this repo.

Comment (stevelilly-youview) [inline snapper-samples/build.gradle]: Doesn't that make this a transitive dependency that would have to be declared somehow?

Comment (chris-yv) [inline snapper-samples/build.gradle]: No? The samples are illustrative, but aren't for inclusion and the library part doesn't depend on the samples.

However, back to the original question, there is a leftover dependency in snapper's `build.gradle` on AppCompat which I'll remove, but for reference it works fine with different versions.

Comment (stevelilly-youview) [inline snapper-samples/build.gradle]: My bad, I was thinking about the snapper library not the samples.

**Comment by stevelilly-youview** [comment]
:+1:

**Pull request merged**